src/components/state.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, ReactNode, useState } from 'react';
+import { Person } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MyContext = createContext<any>(null);
+
+interface MyProvideProps {
+  children: ReactNode;
+}
+
+export const MyProvider = ({ children }: MyProvideProps) => {
```

Comment:
Consider using more descriptive names for your components. Instead of `MyProvider`, you could name it something like `AppContextProvider`.